pkg/model/constants.go

```diff
@@ -44,7 +44,7 @@ const (
 	RouteLoadBalancingStrategy            = "source"
 	IngressDefaultHost                    = "keycloak.local"
 	PostgresqlBackupServiceAccountName    = "keycloak-operator"
-	KeycloakExtensionEnvVar               = "KEYCLOAK_EXTENSIONS"
+	KeycloakExtensionEnvVar               = "KEYCLOAK_EXTENSION"
```

Comment:
I'm a bit concerned here about backward compatibility.

Reply:
There was a breaking change introduced in the upstream image, which I have a strong suspicion was the result of an accident. This change just matches the upstream breaking change.

The alternative would be to fix the change upstream, which is doable as well, if that's preferred.

keycloak/keycloak-containers@da4b38c#diff-51b1ab20d6e750a4464ec80bb5021a47be580d89ccc4cd25427ee812c5893347L33-R44